### PR TITLE
Improve Yaml export for compatibility with Bolt 4/5 Conimex import

### PR DESCRIPTION
**Pull request by bobdenotter**

### CODE REVIEW

File: src/Storage/Migration/Export.php [hunk 1]
```diff
@@ -124,6 +130,7 @@ private function getRecords($contentTypeName, MutableBag $exportData, MutableBag
         $repo = $this->em->getRepository($contentTypeName);
         $metadata = $repo->getClassMetadata();
         // Get all the records for the ContentType
+        // $entities = $this->query->getContent($contentTypeName, ['limit' => 100, 'order' => '-id']);
```
Comment (nestordedios) [inline src/Storage/Migration/Export.php]: I think this comment can be removed. For the rest everything looks good.

Comment (bobdenotter) [reply src/Storage/Migration/Export.php]:
```suggestion
```

Comment (bobdenotter) [reply src/Storage/Migration/Export.php]: Done!

**Review by nestordedios — APPROVED** [review]
Thanks @bobdenotter ! This looks good 👍

**Pull request merged**